src/EORegistryCoordinator.sol

```diff
@@ -474,28 +480,33 @@ contract EORegistryCoordinator is
             emit OperatorRegistered(operator, operatorId);
         }
 
-        // Register the operator with the BLSApkRegistry, StakeRegistry, and IndexRegistry
+        // Register the operator with the EOBLSApkRegistry, EOStakeRegistry, and EOIndexRegistry
         blsApkRegistry.registerOperator(operator, quorumNumbers);
         (results.operatorStakes, results.totalStakes) = 
             stakeRegistry.registerOperator(operator, operatorId, quorumNumbers);
         results.numOperatorsPerQuorum = indexRegistry.registerOperator(operatorId, quorumNumbers);
         if (address(EOChainManager) != address(0)){
-            EOChainManager.registerValidator(operator, results.operatorStakes);
+            if ( chainValidatorSignature.X != 0 && chainValidatorSignature.Y != 0 && chainValidatorG2Signature.X[0] != 0 && chainValidatorG2Signature.X[1] != 0 && chainValidatorG2Signature.Y[0] != 0 && chainValidatorG2Signature.Y[1] != 0){
```

Comment:
add comment
// check if is a valid chain validator signature. if not register as a data validator

Reply:
added